tests/datasets/api_tests.py

```diff
@@ -143,21 +152,22 @@ def test_get_dataset_item(self):
             "cache_timeout": None,
             "database": {"database_name": "examples", "id": 1},
             "default_endpoint": None,
-            "description": None,
+            "description": "Energy consumption",
```

Comment:
side effect from datasource tests, changed this test to used `energy_usage` table that is not modified